compat/test/browser/textarea.test.js

```diff
@@ -23,4 +24,26 @@ describe('Textarea', () => {
 
 		expect(scratch.innerHTML).to.equal('<textarea>foo</textarea>');
 	});
+
+	it('should support resetting the value', () => {
+		let set;
+		const App = () => {
+			const [state, setState] = useState('');
+			set = setState;
+			return <textarea value={state} />;
+		};
+
+		render(<App />, scratch);
+		expect(scratch.innerHTML).to.equal('<textarea></textarea>');
+
+		act(() => {
+			set('hello');
+		});
+		expect(scratch.innerHTML).to.equal('<textarea>hello</textarea>');
```

Comment:
I think this will catch the issue:

```suggestion
		expect(scratch.firstElementChild.value).to.equal('hello');
		expect(scratch.innerHTML).to.equal('<textarea>hello</textarea>');
```